### ISSUE TITLE
`ImmutableList` should serialize nicer

### ISSUE THREAD

**Issue by agroszer** [Shoobx/shoobx.immutable#3]
no need to explicitly serialize ImmutableList, since all lists within an immutable object will be immutables

**Comment by strichter** [comment]
Agreed. Same for dicts. It makes accessing them much harder than it needs to be.
